portfolio/src/main/webapp/scripts/snake/sketch.js

```diff
@@ -0,0 +1,283 @@
+/** Game board */
+let cnv;
```

Comment:
Nit: I think it's more readable actually to be more verbose here (and below) and actually just call this "canvas"

Reply:
That's a good point, a few extra characters never hurt anybody and adding them would definitely make this much more verbose. I pushed a change, hopefully it looks good now!